### PR TITLE
[pvr] fix various issues after introduce #6738

### PR DESCRIPTION
**Pull request by xhaggi**
@Montellese mind taking a look

### CODE REVIEW

**Comment by xhaggi** [comment]
jenkins build this please

**Comment by MilhouseVH** [comment]
Is this fixing this?

http://forum.kodi.tv/showthread.php?tid=211501&pid=1957391#pid1957391

Will add it into my next build.

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -4841,7 +4841,7 @@ std::string CGUIInfoManager::GetItemLabel(const CFileItem *item, int info, std::
     break;
   case LISTITEM_EPISODE:
     {
-      int iSeason, iEpisode;
+      int iSeason = 0, iEpisode = 0;
```
Comment (Montellese) [inline xbmc/GUIInfoManager.cpp]: `iSeason` should probably be initialized to `-1` because `0` means specials (see the check further down).

**Comment by Montellese** [comment]
Looks good apart from the `iSeason` comment.

File: xbmc/GUIInfoManager.cpp (hunk 1 above)
Comment (notspiff) [inline xbmc/GUIInfoManager.cpp]: and episode 0 is also a valid thing.

Comment (xhaggi) [inline xbmc/GUIInfoManager.cpp]: ok will change it to -1

**Comment by xhaggi** [comment]
@Montellese done, please take a second look.

**Comment by xhaggi** [comment]
@MilhouseVH yes

**Comment by Montellese** [comment]
Should be good, thanks.

**Comment by xhaggi** [comment]
thanks.

jenkins build and merge

**Pull request merged**